tests/manual-test-cases/Group18-VIC-UI/18-3-VIC-UI-NGC-tests.robot

```diff
@@ -36,17 +39,21 @@ Check Prerequisites
 
 Ensure Vicui Is Installed
     # ensure vicui is installed before running ngc automation tests
-    Set Vcenter Ip
-    Install Vicui Without Webserver  ${TEST_VC_USERNAME}  ${TEST_VC_PASSWORD}  ${TEST_VC_ROOT_PASSWORD}  ${TRUE}
+    #Install Vicui Without Webserver  ${TEST_VC_USERNAME}  ${TEST_VC_PASSWORD}  ${TEST_VC_ROOT_PASSWORD}  ${TRUE}
+    Update Config To Use Webserver For Plugin Testing
+    Install With Webserver  ${TEST_VC_USERNAME}  ${TEST_VC_PASSWORD}
     ${output}=  OperatingSystem.GetFile  install.log
-    Should Contain  ${output}  was successful
+    ${passed}=  Run Keyword And Return Status  Should Contain  ${output}  was successful
+    Run Keyword Unless  ${passed}  Copy File  install.log  ngc-tests-fail-install-plugin-using-webserver-for-plugin-testing.log
+    Remove File  install.log
+    Should Be True  ${passed}
```

Comment:
After each run of UI nightly test I want any failed test case to have its log ready in the bundle (in this case it would be ngc-tests-fail-blabla.log) Note that I have a different kickoff script that runs only on the VIC-Nightly VM.